lib/camunda-cloud/features/replace/ElementTemplatesReplaceProvider.js

```diff
@@ -1,35 +1,26 @@
-import {
```

Comment:
Proposal: Reuse the full unlink provider which is independent of C8 and C7 details (I think?).

We could then add an additional `ApplyTemplateReplaceBehavior` utility for the C8 casehttps://github.com/camunda/camunda-bpmn-js/pull/219/files#r1047752391.

Or maybe that complicates things? What do you think?

Reply:
> Another option is to have the file shared (in base folder, for example) but not include it in the base modeler and only include it in each C7/C8 modeler. 

Yes, however I'd rather introduce a dedicated `shared` folder with behavior shared across different variations.

Do you want to follow up with that? Looks like this is a matter of personal preference; hence I'd also be fine to merge the PR as is.

Reply:
Exactly :+1: